resources/js/Components/Form/DmEntryCreateForm.tsx

```diff
@@ -64,6 +64,13 @@ const StyledGrid = styled(Grid)`
     margin-bottom: 16px;
 `
 
+const StyledGPTextField = styled(TextField)({
+    '& input::-webkit-clear-button, & input::-webkit-outer-spin-button, & input::-webkit-inner-spin-button':
+        {
+            display: 'none',
+        },
+})
```

Comment:
Hey @AnnieTran13, what is this doing exactly?

Reply:
same question

Reply:
So I looked up how to disable the arrow buttons on a textfield of type number. This is basically just hiding the arrow buttons. The problem with the GP textfield is that we can't remove the type like you did with the other textfields. I tried a lot of different things to keep GP as a float without the type set to number and it just doesn't work.